runtime/nsd_gtls.h

```diff
@@ -99,4 +104,11 @@ rsRetVal gtlsRecordRecv(nsd_gtls_t *pThis);
 #define GTLS_ANON_PRIO "NORMAL:+ANON-DH:+ANON-ECDH:+COMP-ALL"
 #endif
 
+#if GNUTLS_VERSION_MAJOR > 3 || (GNUTLS_VERSION_MAJOR == 3 && GNUTLS_VERSION_MINOR >=3)
+#define EXTENDED_CERT_CHECK_AVAILABLE
+#endif
+
+#define GTLS_KP_SERVER "1.3.6.1.5.5.7.3.1"
```

Comment:
Why not use `GNUTLS_KP_TLS_WWW_SERVER` and `GNUTLS_KP_TLS_WWW_CLIENT` directly?

Reply:
Is it identical functionality-wise? If yes, I will change it, that way it will be more readable for others.

Reply:
Ok, found it at:
https://github.com/gnutls/gnutls/blob/27ec82e781b3960693cd4550440ab7c1664553b8/lib/includes/gnutls/x509.h#L74
will update PR